sdk/src/quotes/public/increase-liquidity-quote.ts

```diff
@@ -109,32 +143,41 @@ export function increaseLiquidityQuoteByInputTokenWithParams(
 
   switch (positionStatus) {
     case PositionStatus.BelowRange:
-      return quotePositionBelowRange(param);
+      return getQuotePositionBelowRange(param);
     case PositionStatus.InRange:
-      return quotePositionInRange(param);
+      return getQuotePositionInRange(param);
     case PositionStatus.AboveRange:
-      return quotePositionAboveRange(param);
+      return getQuotePositionAboveRange(param);
     default:
       throw new Error(`type ${positionStatus} is an unknown PositionStatus`);
   }
 }
 
-/*** Private ***/
+function adjustPriceForSlippage(
```

Comment:
hmm, since sqrtPrice is a square root, I would think that when this value is multiplied by 4, the actual price would move by a factor of 2. 
How about applying slippage to price instead of sqrtPrice or multiplied by the square of slippage ? 🤔